.circleci/config.yml

```diff
@@ -45,6 +45,7 @@ jobs:
             export MYSQL_CONNECTION_USERNAME="root"
             export MYSQL_CONNECTION_PASSWORD="mysql"
             export MONGODB_URL="mongodb://root:mongodb@localhost:27017/admin?ssl=false"
+            export SKIP_RAFT_TESTS="true"
```

Comment:
no need for double quotes here, single quotes are safer in shell if there is no need for variable interpolation, etc.

Reply:
Add a TODO comment referencing the Jira `VAULT-...` that we need to drop this when the related Jira is fixed.

Reply:
Added a comment stating `This will be removed after VAULT-4324 is fixed`